lib/mix/tasks/phoenix.ex

```diff
@@ -0,0 +1,31 @@
+defmodule Mix.Tasks.Phoenix do
+  use Mix.Task
+
+  @shortdoc "Prints Phoenix help information"
+
+  @moduledoc """
+  Prints Phoenix tasks and their information.
+
+    mix phoenix
+
+  """
+
+  @doc false
+  def run(args) do
+    {_opts, args, _} = OptionParser.parse(args)
+
+    case args do
+      [] -> help()
+      _  ->
+        Mix.raise "Invalid arguments, expected: mix phoenix"
+    end
+  end
+
+  defp help() do
+    Application.ensure_all_started(:phoenix)
+    Mix.shell.info "Phoenix v#{Application.spec(:phoenix, :vsn)}"
+    Mix.shell.info "A productive web framework that does not compromise speed and maintainability."
```

Comment:
I thought that at first but it's not very descriptive so decided against it, happy to change if that's preferred though